src/components/Sidebar/NavContent.jsx

```diff
@@ -114,7 +116,7 @@ function NavContent() {
                 </Box>
               </Link> */}
 
-              <Link href="https://app.sushi.com/swap?outputCurrency=0x21585BBcD5bDC3f5737620cf0Db2E51978cf60ac">
+              <Link href="http://inventory.manifest.gg">
```

Comment:
I dont think we should include this link yet, and if we do we should put a temporary coming soon page.